ansible/roles/test/files/ptftests/vxlan-decap.py

```diff
@@ -46,7 +46,8 @@ def count_matched_packets_helper(test, exp_packet, exp_packet_number, port, devi
         raise Exception("%s() requires positive timeout value." % sys._getframe().f_code.co_name)
 
     total_rcv_pkt_cnt = 0
-    while True:
+    end_time = time.time() + timeout
+    while time.time() < end_time:
         result = dp_poll(test, device_number=device_number, port_number=port, timeout=timeout)
```

Comment:
We can also send the `exp_packet` to the `dp_poll()`. That way the result will be `test.dataplane.PollSuccess` only if the expected packet is found by ptf and no need to call `match_exp_pkt()`

Reply:
Great idea. Change in this PR would not cause harm. I feel comfortable checking it in to catch tonight's tests. I'll try your suggest after nightly test is done today and go from there.

Reply:
Done.